services/app/src/components/gameFooter/chess-pieces.css

```diff
@@ -0,0 +1,110 @@
+.flex-center {
```

Comment:
what are all these classess?

Most of them does not seem to be used in this PR so are they needed or added accidentally?